docs/posix-secondaries-bitbaking.adoc

```diff
@@ -0,0 +1,114 @@
+= Posix Secondaries: bitbaking and usage of Primary and Secondary images
+
+The goal of this doc is to guide a reader on bitbaking of two type of images `primary` and `secondary` that are targeted for QEMU (and RPi TBD)
+and running of which on the target makes it act as `Primary` and `Secondary` ECU of a single device.
+
+Please, refer to link:https://uptane.github.io/uptane-standard/uptane-standard.html[the Uptane standard] in order to grok a meaning of the `Primary` and `Secondary` terms in the given context from a theoretical standpoint.
+Please, check out link:posix-secondaries.doc[this doc] to understand these terms from a practical standpoint and
+to learn how `Primary` and `Secondary` can be emulated locally on an user's host.
+It is highly advisable to follow the doc instructions and play with faked/emulated `Primary` and `Secondary` prior to running steps described further in this doc.
```

Comment:
As I mentioned, I am not sure it's warranted to call this setup fake or emulated as it's basically the same as the yocto setup minus the qemu networking part (which is arguably more fake? :smile:)

Maybe we could talk about primary-secondary communication on localhost / single machine network or similar as an introduction and drop most of the "fake" and "emulated" references later in posix-secondaries.adoc?

Reply:
Yeah, I understand your point, not sure if I agree with usage of word 'network' in the term we are looking for.
This doc https://docs.ota.here.com/quickstarts/install-a-client-locally-with-fake-secondaries.html is titled as "Simulate a device on your workstation" so 'simulate device' is used while a link/document is named as 'install-a-client-locally with fake secondaries'.

IMHO, what we do here is simulation of a composite device that includes one primary ECU and one single ECU on a local host (although it can be more than one secondary). Perhaps a term 'multi-ECU device simulation on a local host would do the trick ?

Reply:
>  This doc https://docs.ota.here.com/quickstarts/install-a-client-locally-with-fake-secondaries.html is titled as "Simulate a device on your workstation" so 'simulate device' is used while a link/document is named as 'install-a-client-locally with fake secondaries'.

Yes, but those are virtual secondaries, which are truly and completely fake. These are not that level of fake.